apps/graphql/src/definitions/Build.js

```diff
@@ -200,6 +200,24 @@ export const resolvers = {
         throw new APIError("Invalid user authorization");
       }
 
+      const build = await Build.query()
+        .findById(buildId)
+        .withGraphFetched("repository");
+
+      if (build.repository.private) {
+        const account = await Account.getAccount(build.repository);
+        if (account) {
+          const screenshotsConsumptionRatio =
+            await account.getScreenshotsConsumptionRatio();
+          if (
+            screenshotsConsumptionRatio &&
+            screenshotsConsumptionRatio >= 1.1
+          ) {
+            throw new APIError("Please upgrade to unlock build reviews.");
+          }
+        }
```

Comment:
Without account, we should consider it free. Adding `if (account)` everywhere does not solve it. You have to go with the way of `Account.getAccount` always returning an account, even if this account does not really exist in database.